saltproc/depcode.py

```diff
@@ -159,20 +166,346 @@ def write_mat_file(self, dep_dict, dep_end_time):
         """
 
 
+class DepcodeOpenMC(Depcode):
+    r"""Class contains information about input, output, geometry, and
```

Comment:
Does that leading `r` serve any purpose?

Reply:
`r` makes a raw string. I don't think it's necessary in this case.